src/main/java/org/jabref/gui/preferences/PreferencesDialogViewModel.java

```diff
@@ -174,6 +177,12 @@ public void storeAllSettings() {
             return;
         }
 
+        if (justImported) {
+            dialogService.notify(Localization.lang("Preferences were just imported. No additional changes to save."));
```

Comment:
I think no additional message should be displayed, as nothing special happened. Also that would introduce a new translation string.